packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -57,6 +57,24 @@ const DOM = props => {
 		escapedState,
 	};
 
+	// simple-universal-style-loader accumulates CSS Module style rules
+	// in node's `global` object under the `__styles__` key.
+	//
+	// If this key is populated, we render a single style tag with all
+	// module styles needed for the request.
+	let moduleStyleTag;
```

Comment:
even better, define a function called `getModuleStyleTags` outside of the `Dom` class to avoid the `let` and encapsulate this behavior - I would potentially even break this out into a standalone utility module, but that's probably overkill in terms of separating concerns.

Another alternative is to follow the same pattern that `cssLinks` is using and pass in an array of `style` strings as a `prop` to `Dom` that can be mapped to `<style>` tags internally - that might be nice to separate the _ability_ to inject `style` tags from the specific way that `simple-universal-style-loader` _defines_ the styles - make it the responsibility of the function that calls `<Dom>` rather than `Dom`'s direct responsibility.

```jsx
const { styles = [] } = this.props;

...
<head>
  {styles.map((body, key) => <style type="text/css" key={key}>body</style>}
</head>
```

```jsx
const { getStyles } = require('simple-universal-style-loader');
...
<Dom styles=`getStyles()` ... />
```